spec/dummy/app/controllers/cookies_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class CookiesController < ApplicationController
+  include ActionController::Cookies
+  protect_from_forgery with: :null_session
```

Comment:
WDYT about doing what we did in https://github.com/graphql-devise/graphql_devise/blob/74b5f2a83e65385bdf843fb6ac3cbe6fec88efbe/spec/dummy/app/controllers/api/v1/graphql_controller.rb#L46 instead of setting a null session here? That way we could probably test that the correct cookie is set after login and other mutations